raster-cli/public/ImageFactory.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <filesystem>
+#include <fstream>
+
+/**
+ * Reads a type ID from the first whitespace separated string in file.
+ *
+ * @param file_path [in] Path to file.
+ * @returns The format ID of @ref file_path.
+ */
+std::string load_format_id(const std::string& file_path) {
```

Comment:
This function is into the header, so, it must be static

Reply:
Good point! Thanks!